packages/block-template-html/package.json

```diff
@@ -23,7 +22,7 @@
     "dev": "echo \"Serving at http://localhost:63212\"; http-server -p 63212 ./dev",
```

Comment:
The build script is deleting the `block-schema.json`, `block-metadata.json`, `src` and `dev` folders. Is this intended?

Reply:
Which build script do you refer to?

Reply:
the one in the html template package.json: `"build": "rimraf dist; cp -R src dist; cp block-metadata.json dist; cp block-schema.json dist; cp -R public dist;"`

Reply:
Just tried `yarn workspace block-template-html build` in PowerShell on macOS — all files seem to be present.

Reply:
Can confirm it works well on macOS

Reply:
I ran it again to make sure and it is still removing those files. Ran it with WSL and it's working fine so it should be a problem with cmd. Will approve it and create a task to track this.

Reply:
Here it is: https://app.asana.com/0/1201095311341924/1202768723281652/f